pkg/stub/handler.go

```diff
@@ -6,25 +6,19 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"net"
+	"net/http"
 	"os"
 	"runtime"
+
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/sirupsen/logrus"
-
-	corev1 "k8s.io/api/core/v1"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
-	kapis "k8s.io/apimachinery/pkg/apis/meta/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	utilnet "k8s.io/apimachinery/pkg/util/net"
-	"k8s.io/apimachinery/pkg/util/wait"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
-	corev1lister "k8s.io/client-go/listers/core/v1"
-	restclient "k8s.io/client-go/rest"
-	"k8s.io/client-go/util/retry"
+	"golang.org/x/mod/semver"
+	"gopkg.in/yaml.v2"
```

Comment:
The following links may be useful

* https://github.com/kubernetes-sigs/yaml
* https://pkg.go.dev/sigs.k8s.io/yaml

Reply:
done